edge/pkg/devicetwin/dtmanager/communicate_test.go

```diff
@@ -0,0 +1,320 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dtmanager
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/kubeedge/kubeedge/common/beehive/pkg/core/context"
```

Comment:
this import path doesn't exist anymore. Please change

Reply:
Updated